recipes/libmatio/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.5.6" %}
+package:
+  name: libmatio
+  version: {{ version }}
+
+source:
+  fn: matio-{{ version }}.tar.gz
+  url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
+  md5: bb53fd06f7ebb74589e9dd684b1b66a6
+
+build:
+  number: 0
+  skip: true     # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+    - hdf5
+    - zlib
+
+  run:
+    - hdf5
+    - zlib
+
+test:
+  commands:
+    - test -e $PREFIX/include/matio.h  # [unix]
```

Comment:
lol, @pelson said the same thing in another issue. There is some jinja logic that I wrote for Boost that is generic enough that we could put it in some file and import it with jinja. However, I think there are a few features (like selectors) that don't work with conda build jinja imports yet. Probably should be put in a new issue until it can find a home. Maybe I opened one already? :laughing: I'll search for it.